src/components/CurrentRefinements/CurrentRefinements.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'preact-compat';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import upperFirst from 'lodash/upperFirst';
+import Template from '../Template.js';
+import { isSpecialClick } from '../../lib/utils.js';
+
+function handleClick(cb) {
```

Comment:
irrelevant for this PR, but seems like handleClick & isSpecialClick are always used together. It should be a single function "avoidSpecialClick" instead IMO